### PR TITLE
docs: update readme to include recommended versions

### PR DESCRIPTION
**Pull request by SWvheerden**
Description
---
Update readme to include recommended versions for each network.

Breaking Changes
---

- [x] None
- [ ] Requires data directory on base node to be deleted
- [ ] Requires hard fork
- [ ] Other - Please specify

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,13 @@ Only the first four applications will be discussed in this README (see [wallet-a
 
 ## Installing using binaries
 
+### Versions
+For  the recommended versions, for more detail about versioning see [Release Ideology](https://github.com/tari-project/tari/blob/development/docs/src/branching_releases.md)
+
+| Network       | StageNet | NextNet | Develpment |
```
Comment (stringhandler) [inline README.md]: nit: I prefer the network to be rows

Comment (SWvheerden) [reply README.md]: I have no pref, let me invert the table

**Review by stringhandler — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -14,6 +14,15 @@ Only the first four applications will be discussed in this README (see [wallet-a
...
 ## Installing using binaries
 
+### Versions
+For  the recommended versions, for more detail about versioning see [Release Ideology](https://github.com/tari-project/tari/blob/development/docs/src/branching_releases.md)
+
+| Network  | Version      | 
+|----------|--------------|
+| Stagenet | 0.45.0       |
+| NextNet  | 0.48.0-rc.2  |
+| Develpment | 0.49.0-pre.4 |
```
Comment (stringhandler) [inline README.md]:
```suggestion
| Development | 0.49.0-pre.4 |
```

File: README.md [hunk 3]
```diff
@@ -14,6 +14,15 @@ Only the first four applications will be discussed in this README (see [wallet-a
 
 ## Installing using binaries
 
+### Versions
+For  the recommended versions, for more detail about versioning see [Release Ideology](https://github.com/tari-project/tari/blob/development/docs/src/branching_releases.md)
+
+| Network  | Version      | 
+|----------|--------------|
+| Stagenet | 0.45.0       |
+| NextNet  | 0.48.0-rc.2  |
```
Comment (stringhandler) [inline README.md]:
```suggestion
| Nextnet  | 0.48.0-rc.2  |
```

**Review by stringhandler — APPROVED** [review]

**Comment by stringhandler** [comment]
ACK

**Pull request merged**